docs/execution-analysis/gpu_acceleration.md

```diff
@@ -1,29 +1,59 @@
 # GPU acceleration
 
+This document explains how to use GPU accelerations with Concrete.
+
 Concrete supports acceleration using one or more GPUs.
 
-To use this feature, you need to install a Concrete Python wheel which is built with GPU/CUDA support. This is **not** available on pypi.org/project/concrete-python as we only release wheels with CPU support there. To install a GPU/CUDA wheel, you need to install it from our [Zama public pypi repository](https://pypi.zama.ai), which can be done with the following command line:
+{% hint style="info" %}
+This version is not available on [pypi.org](pypi.org/project/concrete-python), which only hosts wheels with CPU support. 
+{% endhint %}
+
+To use GPU acceleration, install the GPU/CUDA wheel from our [Zama public PyPI repository] (https://pypi.zama.ai) using the following command:
 
 `pip install concrete-python  --index-url https://pypi.zama.ai/gpu`.
 
-Once a GPU/CUDA flavor python wheel is installed, FHE program compilation must be [configured](../guides/configure.md) using the **use_gpu** option to enable GPU offloading.
+After installing the GPU/CUDA wheel, you must [configure] ((../guides/configure.md)) the FHE program compilation to enable GPU offloading using the `use_gpu` option.
 
 {% hint style="info" %}
-Our GPU wheels are built with CUDA 11.8 and should be compatible with higher versions of CUDA.
+Our GPU wheels are built with CUDA 11.8 and are compatible with higher versions of CUDA.
```

Comment:
not sure are is the best wording here as we cannot ensure cuda version will be backward compatible with 11.8, so I think should be is more true. (cc @antoniupop )

Reply:
Yes, indeed that was the original rationale for the wording. It is very likely that most higher versions are compatible with the wheel, but we can't guarantee that!

Reply:
We have not tested with other CUDA, and it would be very complicated to do so

Reply:
We have tested with a few, yes - at least I've tried 12.2 and 12.3 myself so far. I can't test them all, but I would think that Nvidia have mostly kept things compatible so newer drivers can execute codes compiled against older libs.